share_extension/ios/extension_post.js

```diff
@@ -42,6 +44,7 @@ import ExtensionTeams from './extension_teams';
 const ShareExtension = NativeModules.MattermostShare;
 const MAX_INPUT_HEIGHT = 95;
 const MAX_MESSAGE_LENGTH = 4000;
+const MAX_FILE_SIZE = 20 * 1024 * 1024;
```

Comment:
Should we implement the same max file size limitation on Android share extension?